src/capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './decorator'
```

Comment:
> - [ ] Please I suggest you use require_relative instead of require as the file path throws an error when checking the result.👍
> 
> ![tu1](https://github.com/fatima-najafi/oop-school-library/assets/40009354/2e7ea7f6-b8c6-4a95-90cd-4fd9a3f1012f)
> 
> ```suggestion
> require_relative 'decorator'
> ```